include/qemu/int128.h

```diff
@@ -373,11 +383,21 @@ static inline bool int128_ge(Int128 a, Int128 b)
     return a.hi > b.hi || (a.hi == b.hi && a.lo >= b.lo);
 }
 
+static inline bool int128_uge(Int128 a, Int128 b)
+{
+    return (uint64_t)a.hi > (uint64_t)b.hi || (a.hi == b.hi && a.lo >= b.lo);
+}
+
 static inline bool int128_lt(Int128 a, Int128 b)
 {
     return !int128_ge(a, b);
 }
 
+static inline bool int128_ltu(Int128 a, Int128 b)
```

Comment:
```suggestion
static inline bool int128_ult(Int128 a, Int128 b)
```

Reply:
I forgot to push the fixup for this typo. The actual branch is based on your last vdiv/vmod patch series, so this will actual go away in the final version.